src/components/Dashboard/dashboard.module.scss

```diff
@@ -0,0 +1,77 @@
+.dashboard { 
+    border-left: #dad8d8 solid 2px;
+    border-radius: 3px;
+    padding-left: 3%;
+    margin: 1%;
+}
+.dashboardContainer {
+    overflow-y: scroll;
+    overflow-x: hidden;
+    height: 800px;
+}
+
+.dashboardNav { 
+    display: flex;
+    justify-content: space-between;
+    --width: 100%;
+    padding: 1%;
+}
+
+.dashboardReportCard { 
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 1rem;
+    border: #dad8d8 solid 1px;
+    border-radius: 3px;
+    padding: 3%;
+}
+
+.dashboardTitle { 
+    width: 50%;
+    padding-top: 20px;
+    display: grid;
+}
+.dashboardScore { 
+    display: inline-flex;
+    background-color: var(--seed-color-primary);
+    border-radius: 50%;
+    padding: 10px;  
+    text-align: center;
+    margin: 8px;
+    color: #ffeaea;
+}
+
+.dashboardStats { 
+    display: block;
+}
+
+.dashboardCategoryTitle { 
+    width: 100%;
+    padding: 1%;
+    --padding-top: 3%;
+}
+
+.dashboardContainer::-webkit-scrollbar {
+    width: 10px;
+}
+   
+.dashboardContainer::-webkit-scrollbar-thumb {
+    background: rgb(206, 205, 205);
+    border-radius: 20px;
+}
+  
+.dashboardContainer::-webkit-scrollbar-track {
+    background:#f1f1f1;
+    border-radius: 20px;
+}
```

Comment:
substitute the px for rems

Reply:
see if you can use a color we already have instead of ```#f1f1f1``` else make a new CSS variable